src/common.js

```diff
@@ -137,6 +137,35 @@ Common.prototype.sendConsent = function (type, payload) {
     customDataLayer('consent', type, payload);
 };
 
+Common.prototype.getEventConsentState = function (eventConsentState) {
+    return eventConsentState && eventConsentState.getGDPRConsentState
+        ? eventConsentState.getGDPRConsentState()
+        : {};
+};
+
+Common.prototype.maybeSendConsentUpdateToGoogle = function (consentState) {
+    // If consent payload is empty,
+    // we never sent an initial default consent state
+    // so we shouldn't send an update.
+    if (this.consentPayloadAsString && this.consentMappings) {
+
+        if (!this.isEmpty(consentState)) {
```

Comment:
I can probably move this to the if statement at 150 to avoid unnecessary nested if statement